WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -90,6 +92,20 @@ - (void)fetchPostsFromEndpoint:(NSURL *)endpoint
     [self fetchPostsFromEndpoint:endpoint withParameters:params success:success failure:failure];
 }
 
+- (void)fetchPostsFromEndpoint:(NSURL *)endpoint
+                         count:(NSUInteger)count
+                        offset:(NSUInteger)offset
+                       success:(void (^)(NSArray<RemoteReaderPost *> *))success
+                       failure:(void (^)(NSError *))failure
+{
+    NSDictionary *params = @{@"number": @(count),
+                             @"offset": @(offset),
+                             @"order": @"DESC",
```

Comment:
(For another PR) this makes me think we should normalize the keys. `DESC` seems a pretty common parameter